### PR TITLE
v6.0.0

### PR DESCRIPTION
**Pull request by trentm**
Drop node v6 support.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2016-2019 Thomas Watson Steen
```
Comment (watson) [inline LICENSE:3]: Nit to align with the copyright format below

```suggestion
Copyright (c) 2016-2019, Thomas Watson Steen
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+- Drop node v6 support. New minimum supported node version is v8.6.0.
+  (This is the min support node version for elastic-apm-node@3 that uses
+  this module.)
+- Add testing of node v19.
+- Rename default branch from "master" to "main".
+- Should there be a need to do 5.x maintenance releases there is a
+  [5.x branch](https://github.com/elastic/require-in-the-middle/tree/5.x).
+
+
```
Comment (watson) [inline CHANGELOG.md:13]: Just personal formatting nits. Feel free to ignore 😅 

```suggestion
- Drop Node.js 6 support. New minimum supported Node.js version is 8.6.0.
  (This is the minimum supported Node.js version for elastic-apm-node@3 that uses
  this module.)
- Add testing of Node.js 19.
- Rename default branch from `master` to `main`.
- Should there be a need to do 5.x maintenance releases there is a
  [5.x branch](https://github.com/elastic/require-in-the-middle/tree/5.x).

```

**Review by watson — APPROVED** [review]
LGTM 👍 I just left a few suggestions/nits for some formatting, but otherwise all good!

**Comment by trentm** [comment]
require-in-the-middle@6.0.0 published
v6.0.0 tagged

**Pull request merged**